Sources/Common/Extensions/String.swift

```diff
@@ -119,6 +119,18 @@ extension String {
     public func replaceNameTemplate(with name: String) -> String {
         return self.replacingOccurrences(of: "\\{.*?\\}", with: name, options: .regularExpression)
     }
+    
+    /// Return package name from a full api file name.
+    /// For example:
+    /// self = "/users/username/swagger/products/api.yaml
+    /// root = "/users/username/swagger"
+    /// returns "products"
+    public func getPackageName(root: String) -> String {
```

Comment:
And you wrote tests on it, didn't you? (:

Reply:
Yep :)